src/lib/model.ts

```diff
@@ -17,7 +17,10 @@ export class Course {
 
 		course.modules = (data.modules || []).map(Module.create)
 
-		const audiences = (data.audiences || []).map(Audience.create)
+		let audiences = []
+		if (data.course && data.course.audiences) {
+			audiences = (data.course.audiences).map(Audience.create)
+		}
```

Comment:
this looks wrong to me - audiences are at the same level as modules - so it should be they either both come from `data.course` or both from `data` - like it was originally

is there a test that could demonstrate this fix is required that way?